Formula/dita-ot.rb

```diff
@@ -0,0 +1,21 @@
+class DitaOt < Formula
+  desc "DITA Open Toolkit is an implementation of the OASIS DITA specification"
+  homepage "https://www.dita-ot.org/"
+  url "https://github.com/dita-ot/dita-ot/releases/download/3.1/dita-ot-3.1.zip"
+  sha256 "256163beed7958218653052f9cb8dee43a393f83f6d6c36e5f9d1cc3f29da93b"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8+"
+
+  def install
+    rm_f Dir["bin/*.bat", "startcmd.*"]
+    rm_rf "temp"
+    libexec.install Dir["*"]
+    bin.install_symlink libexec/"bin/dita" => "dita"
+  end
+
+  test do
+    assert_match "DITA-OT version #{version}", shell_output("#{bin}/dita --version")
```

Comment:
would like to see a functional test here

Reply:
Done, it's now parsing the documentation that's part of every release.